src/hotspot/share/jvmci/vmStructs_jvmci.cpp

```diff
@@ -215,6 +216,10 @@
   nonstatic_field(JavaThread,                  _poll_data,                                    SafepointMechanism::ThreadData)        \
   nonstatic_field(JavaThread,                  _stack_overflow_state._reserved_stack_activation, address)                            \
   nonstatic_field(JavaThread,                  _held_monitor_count,                           int64_t)                               \
+  JVMTI_ONLY(nonstatic_field(JavaThread,       _is_in_VTMS_transition,                        bool))                                 \
+  JVMTI_ONLY(nonstatic_field(JavaThread,       _is_in_tmp_VTMS_transition,                    bool))                                 \
+                                                                                                                                     \
+  static_field(JvmtiVTMSTransitionDisabler,    _VTMS_notify_jvmti_events,                     bool)                                  \
```

Comment:
I think this should be also under `JVMTI_ONLY()`.

Reply:
No. Thought it was accessible without JVMTI. Thanks for pointing this out, I have pushed an update.